scripts/globals/spells/trust/monberaux.lua

```diff
@@ -14,16 +14,30 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    -- TODO: Summon: I am Doctor Monberaux. My services are available for any affliction.
     -- TODO: Summon (1 dose of Final Elixir ready): I received a vial of a most valuable medicine. It should prove useful in times of emergency.
     -- TODO: Summon (2 doses of Final Elixir ready): I received two vials of a most valuable medicine. They should prove useful in times of emergency.
     xi.trust.message(mob, xi.trust.message_offset.SPAWN)
 
-    -- local healingMoveCooldown = 10
+    local healingMoveCooldown = 3
```

Comment:
This was a placeholder, did you test this delay on retail?

Reply:
I referenced BGWiki again for this, which says that he uses an ability every 3-4 seconds as needed.

Reply:
we should check in game, as that is someone going "its about such and such" without measuring or they wouldn't say "about" nor "3-4" they'd give an exact value.

Reply:
From my experience on retail, it looks to be 3 seconds on the dot. At least judging by the in game combat log.

Reply:
likely whatever multiple of the battle tick rate amounts to 3 so likely within mere milliseconds of 3.